pyaedt/edb_core/stackup.py

```diff
@@ -1210,6 +1210,36 @@ def place_a3dcomp_3d_placement(self, a3dcomp_path, angle=0.0, offset_x=0.0, offs
 
         return True
 
+    @pyaedt_function_handler
+    def report_copper_rate(self):
```

Comment:
Maybe a better and self explaining name would be better.
get_copper_rate_per_layer? or something similar

Reply:
Change to residual_copper_area_per_layer().